### PR TITLE
refactor: reorganize frontend and switch to Vuetify

### PR DESCRIPTION
**Pull request by milana-todorovic**
The following changed were made to the frontend:
- switched from VueMaterial to Vuetify
- routes reorganized to reflect the hierarchal structure of pages
- separated pages from components (Navbar can be used as reference for how to add a new reusable component (with props), and login and registration pages can be used as a reference for how to add a new page).

### CODE REVIEW

File: frontend-vue-js/.env [hunk 1]
```diff
@@ -1,2 +1,4 @@
-VUE_APP_APIUSERSURL=http://localhost:8081/api/users
-BACKEND_APP_URL = http://localhost:8081/
\ No newline at end of file
+VUE_APP_USER_API_ENDPOINT = api/users
+
+VUE_APP_LOGIN_ENDPOINT = auth/login/
+VUE_APP_REGISTRATION_ENDPOINT = auth/signup/
```
Comment (vlaksi) [inline frontend-vue-js/.env:4]: Will we put all endpoints here? And just use them in app? If yes, then we need to inform rest of the our team.

Comment (Pufke) [reply frontend-vue-js/.env:4]: Don't worry baby 😎

Comment (Pufke) [reply frontend-vue-js/.env:4]: It is same aproach. We use .env conf file before repacking

Comment (vlaksi) [reply frontend-vue-js/.env:4]: I now, but we did not make a agreement about what we would put here. But if we will put all endpoints, it is nice for me.

**Comment by Pufke** [comment]
For now everything looks fine.

**Review by Pufke — APPROVED** [review]

**Comment by Pufke** [comment]
If you want merge PR, and continue authorization on other PR.

**Pull request merged**